code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/AEPMessage.java

```diff
@@ -59,8 +59,9 @@ class AEPMessage implements FullscreenMessage {
     int baseRootViewWidth;
     int frameLayoutResourceId = 0;
     final MessagesMonitor messagesMonitor;
-    FullscreenMessageDelegate fullScreenMessageDelegate;
+    FullscreenMessageDelegate listener;
     MessageFragment messageFragment;
+    MessagingDelegate messagingDelegate = ServiceProvider.getInstance().getMessageDelegate();
```

Comment:
Can you remove this member and retrieve ServiceProvider.getInstance().getMessageDelegate() when needed?